azkaban-solo-server/build.gradle

```diff
@@ -4,6 +4,8 @@ dependencies {
   compile(project(':azkaban-common'))
```

Comment:
Since the plugin uses slf4j library, we need to have sl4j-log4j library to let slf4j work.